src/python/dxpy/bindings/apollo/dataset.py

```diff
@@ -0,0 +1,120 @@
+from dxpy import DXHTTPRequest
+from dxpy.bindings import DXRecord
+
+
+class Dataset(DXRecord):
+    def __init__(self, record_id, project_id):
+        self.record_id = record_id
+        self.project_id = project_id
+        self.visualize_info = None
+        self._dx_record_obj = None
+        self.dx_dataset_descriptor = None
+        self.assays_list = None
+        self.assay_names_list = None
+
+    def get_visualize_info(self):
+        if self.visualize_info is None:
+            self.visualize_info = DXHTTPRequest(
+                "/" + self.record_id + "/visualize",
+                {"project": self.project_id, "cohortBrowser": False},
+            )
+        return self.visualize_info
+
+    @property
+    def vizserver_url(self):
+        vis_info = self.get_visualize_info()
+        return vis_info.get("url")
+
+    def get_dx_record(self):
```

Comment:
If you use the initialize the superclass in `__init__` then you can get rid of this